app/scrapers/__init__.py

```diff
@@ -42,6 +42,10 @@ def feed_gen(query, engine, count=10, qtype=''):
     engine = old_names.get(engine, engine)
     if engine in ('quora', 'youtube'):
         urls = scrapers[engine].search_without_count(query)
+    elif (engine in ['bing']) and (qtype == 'vid'):
```

Comment:
It would be helpful if we follow the same convention for conditional statements. As line 43 uses a tuple, we should use tuple for line 45 and 47 as well, i.e. changing `engine in ['bing']` to `engine in ('bing')`.

Reply:
`elif engine in ['bing'] and qtype == 'vid':`  # Lose the extra parens -- this is not JavaScript

Reply:
I was talking about the use of list and tuple in the conditional statements. We should follow one convention to reduce confusion and better readability.